src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -29,6 +29,16 @@ class DropDownGroup extends React.Component {
     document.addEventListener("click", this.handleOutsideClick);
   }
 
+  static getDerivedStateFromProps(props, state) {
+    if (props.isOpen !== state.isOpenPrevProp) {
```

Comment:
Do we need state.isOpenProvProp? Couldn’t we compare props.isOpen !== state.isOpen?

Reply:
Yes, we do. `state.isOpen` is managed from inside the component. And it doesn't change when we pass a new `props.isOpen` from the parent component. That's why, when we want to force close the DropDown (which is our case, when we have custom handlers for DropDownOptions), we need to look at the `props.isOpen` and compare it with previous one and give it a priority in the case when it has changed.